shared/util/arrays.js

```diff
@@ -16,3 +16,7 @@ export function intersperseFn (separatorFn: (index: number, x: any) => any, arr:
   }, [arr[0]])
 }
 
+export function filterNull<X> (arr: Array<?X>): Array<X> {
+  // $FlowIssue can't understand this
+  return arr.filter(i => !!i)
+}
```

Comment:
Cool! I'll stick with this for now since it's typed and simple, but maybe with enough of these we'll start typing out a subset of lodash.